src/shared/components/app-icon/AppIcon.scss

```diff
@@ -0,0 +1,61 @@
+.appIcon {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  height: var(--app-icon-size, 32px);
+  width: var(--app-icon-size, 32px);
+  background-color: var(--app-icon-bg-color, transparent);
+
+  svg {
+    fill: white;
+  }
+}
+
+.globalSearch {
+  svg {
+    width: 62.5%;
```

Comment:
why are we using %? why are we not specifying the width in px? % should be avoided isn't it?

Reply:
> why are we not specifying the width in px?

Because what I am after is the ratio between the size of the `.iconWrapper` class and the size of the svg inside of it. This allows the svg to respond to the dimensions of the `iconWrapper`. Consider the help page, where the same icons are rendered at different sizes (smaller in the grey panel below): 

![image](https://user-images.githubusercontent.com/6834224/148795396-128feedb-e26d-443b-b994-806a3074f471.png)